src/System.Diagnostics.Process/tests/ProcessTests.netcoreapp.cs

```diff
@@ -40,5 +36,75 @@ public void Start_StandardInputEncodingPropagatesToStreamWriter()
 
             Assert.Same(encoding, process.StandardInput.Encoding);
         }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void StartProcessWithArgumentList()
+        {
+            ProcessStartInfo psi = new ProcessStartInfo(GetCurrentProcessName());
+            psi.ArgumentList.Add("arg1");
+            psi.ArgumentList.Add("arg2");
+
+            Process testProcess = CreateProcess();
+            testProcess.StartInfo = psi;
+
+            try
+            {
+                testProcess.Start();
+                Assert.Equal(string.Empty, testProcess.StartInfo.Arguments);
+            }
+            finally
+            {
+                if (!testProcess.HasExited)
+                    testProcess.Kill();
+
+                Assert.True(testProcess.WaitForExit(WaitInMS));
+            }
+        }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void StartProcessWithSameArgumentList()
+        {
+            ProcessStartInfo psi = new ProcessStartInfo(GetCurrentProcessName());
+            psi.ArgumentList.Add("arg1");
+            psi.ArgumentList.Add("arg2");
+
+            Process testProcess = CreateProcess();
+            Process secondTestProcess = CreateProcess();
+            testProcess.StartInfo = psi;
+            try
+            {
+                testProcess.Start();
+                Assert.Equal(string.Empty, testProcess.StartInfo.Arguments);
+                secondTestProcess.StartInfo = psi;
+                secondTestProcess.Start();
+                Assert.Equal(string.Empty, secondTestProcess.StartInfo.Arguments);
+            }
+            finally
+            {
+                if (!testProcess.HasExited)
+                    testProcess.Kill();
+
+                Assert.True(testProcess.WaitForExit(WaitInMS));
+
+                if (!secondTestProcess.HasExited)
+                    secondTestProcess.Kill();
+
+                Assert.True(testProcess.WaitForExit(WaitInMS));
+            }
+        }
+
+        [Fact]
+        public void BothArgumentAndArgumentListSet()
+        {
+            ProcessStartInfo psi = new ProcessStartInfo(GetCurrentProcessName(), "arg3");
+            psi.ArgumentList.Add("arg1");
+            psi.ArgumentList.Add("arg2");
+
+            Process testProcess = CreateProcess();
+            testProcess.StartInfo = psi;
+            Assert.Throws<InvalidOperationException>(() => testProcess.Start());
+        }
```

Comment:
What about a test for when I don't pass arguments in the PSI, but I do set both PSI.Arguments  and PSI.ARgumentList. That should also throw IOE.